src/controllers/supportDetails.js

```diff
@@ -20,9 +20,7 @@ exports.get = (req, res, next) => {
 exports.put = (req, res, next) => {
   const data = req.body;
   updateTicket(data)
-    .then((response) => {
-      res.send(data);
-    })
+    .then(res.send(data))
```

Comment:
this needs to be a callback.
inside the ```.then()``` will always fire if it's not inside a callback. Even if it's an error.